docs/reference.md

```diff
@@ -212,7 +212,6 @@ The following packages are available to be used in the Node.js 6.9.1 environment
 - [cheerio v0.22.0](https://www.npmjs.com/package/cheerio) - Fast, flexible & lean implementation of core jQuery designed specifically for the server.
 - [cloudant v1.6.2](https://www.npmjs.com/package/cloudant) - This is the official Cloudant library for Node.js.
 - [commander v2.9.0](https://www.npmjs.com/package/commander) - The complete solution for node.js command-line interfaces.
-- [consul v0.27.0](https://www.npmjs.com/package/consul) - A client for Consul, involving service discovery and configuration.
```

Comment:
Why touch this? It's a package installed in our Node.js runtime, which has nothing to do with our system's consul.